src/Spago/Command/Fetch.purs

```diff
@@ -377,7 +378,12 @@ getPackageDependencies packageName package = case package of
       Right { yaml: { package: Just { dependencies: (Dependencies deps) } } } -> do
         pure (Just (map (fromMaybe Config.widestRange) deps))
       Right _ -> die [ "Read valid configuration from " <> configLocation, "However, there was no `package` section to be read." ]
-      Left err -> die [ "Could not read config at " <> configLocation, "Error: " <> err ]
+      Left errLines -> die
+        [ Log.text $ "Could not read config at " <> configLocation
+        , Log.text "Error: "
+        -- type annotation needed here to clarify that this is `Doc GraphicsParam`, not `Doc a`
+        , Log.indent $ Log.lines $ map Log.text errLines :: Docc
+        ]
```

Comment:
The rest of the codebase uses `toDoc` rather than things like `Dodo.text`, I think it works better for inference?

Also note that we are re-exporting `indent` from the Prelude, so most of the times you don't need to import Dodo at all

Reply:
Ah, that's what I needed to do. Thanks. This change was largely just to fix the inference stuff.